ISORROPIA/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ target_compile_options(Isorropia PRIVATE
   ""
   # NOTE: ISORROPIA probably needs to be compiled
   # with REAL*8 or else it'll crash
-  $<$<STREQUAL:"${CMAKE_Fortran_COMPILER_ID}","Intel">:-r8>
```

Comment:
This is required for the Intel 2021.06.0 compiler to understand Fortran 77 files.